cpp/ql/src/semmle/code/cpp/ir/dataflow/DefaultTaintTracking.qll

```diff
@@ -177,8 +221,7 @@ private predicate instructionTaintStep(Instruction i1, Instruction i2) {
   // together in a single virtual variable.
   // TODO: Will this work on the test for `TaintedPath.ql`, where the output arg
   // is a pointer addition expression?
-  i2 =
-    any(WriteSideEffectInstruction outNode |
+  i2 = any(WriteSideEffectInstruction outNode |
```

Comment:
This change might be because I haven't updated my autoformatter.  I'll see if I can update it.

Reply:
Fixed.